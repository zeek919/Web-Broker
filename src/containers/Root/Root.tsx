import React from 'react';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import Login from '../../components/Login/Login';
import GlobalStyle from './GlobalStyle';

const Root = () => (
    <Router>
        <GlobalStyle />
        <Switch>
            <Route exact path="/">
                <Login />
            </Route>
        </Switch>
    </Router>
);
export default Root;
