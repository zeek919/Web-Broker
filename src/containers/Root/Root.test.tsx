import React from 'react';
import Root from './Root';

describe('test', () => {
    it('still testing', () => {
        expect(<Root />).toMatchSnapshot();
    });
});
