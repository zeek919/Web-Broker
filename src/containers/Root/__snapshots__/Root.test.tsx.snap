// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test still testing 1`] = `<Root />`;
